docs/context-variables.md

```diff
@@ -128,6 +134,45 @@ automations:
     {{ branch.diff.files_metadata | match(attr='file', regex=r/\.md$/) | every }}
     ```
 
+#### `env` :fontawesome-solid-flask:  :fontawesome-brands-github:
+
+The `env` context allows the user to pass data from the repo unavailable in the other context variables. Thus, the structure of the variable is not fixed and depends on user configuration.
+
+To configure the `env` variable, add the `env` field to gitstream's workflow job configurations in `gitstream.yml`, under GitHub's `workflows` directory
+
+```yaml title="examle: add secrets to the env variable"
+...
+jobs:
+  gitStream:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    name: gitStream workflow automation
+    env:
+      JIRA_TOKEN: ${{ secrets.JIRA_TOKEN }}
+      SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+    steps:
+...
+```
+
+To use the context variable, access to the `env` variable's fields as configured in `gitstream.yml`
+
+```yaml+jinja title="examle: use slack webhook secret"
+automations:
+  slack_message:
+    if:
+      - true
+    run:
+      - action: http-request@v1
+        args:
+          url: {{ slack.base }}{{ slack.channel }}
+          method: POST
+          headers: '{"Content-type": "application/json"}'
+          body: '{"text": "Hello, world!"}'
```

Comment:
Something that would help me generate examples is to see how to post the PR title and a link to the PR in the slack message. Can you modify this example to demonstrate that? I also think that would be a good starting point for users who are first learning how to use this.